src/theme/index.tsx

```diff
@@ -85,7 +85,7 @@ function getSettings(darkMode: boolean) {
     deprecated_mediaWidth: deprecated_mediaWidthTemplates,
 
     navHeight: 72,
-    mobileBottomBarHeight: 52,
+    mobileBottomBarHeight: 70,
```

Comment:
```suggestion
    mobileBottomBarHeight: 68,
```

`72` would also work here

It's not documented well, but we try to keep all px values to increments of `4`

Reply:
Jordan's right about us trying to use a 4px base. But this should be even taller @ `76` to have more spacing between the tab labels and the home screen bar.